src/xrGame/inventory_item.cpp

```diff
@@ -29,6 +29,10 @@
 
 #define ITEM_REMOVE_TIME 30000
 
+constexpr pcstr invNameKey = "inv_name";
```

Comment:
1) Constants are usually named using CAPITAL letters (remember about misleading)
2) I'd prefer to use constexpr const pcstr here.

Reply:
2. We can create special type alias for this case: `using cpcstr = const char* const;`

Reply:
@FreeZoneMods 
2. `pcstr` is `const char*`. It will be `constexpr const const char*`. Does that make sense?

Reply:
2. `constexpr` implies `const`. I don't see the need for a change here.